docs/source/development/add-ons/install-an-add-on-dev-prerelease.md

```diff
@@ -0,0 +1,126 @@
+---
+myst:
+  html_meta:
+    "description": "How to install an add-on in development mode in Volto (development or pre-release) in your Plone project"
+    "property=og:description": "How to install an add-on in development mode in Volto (development or pre-release) in your Plone project"
+    "property=og:title": "Install an add-on in development mode in Volto (development or pre-release)"
+    "keywords": "Volto, Plone, add-on, pre-release, development, mode"
+---
+
+# Install an add-on in development mode in Volto (development or pre-release)
+
+Use [`mrs-developer`](https://www.npmjs.com/package/mrs-developer) to manage the development cycle of Volto add-ons.
+This tool pulls the remote code and configures the current project, making the add-on available for the build.
+By doing this, you can develop both the project and the add-on product as if they were both part of the current codebase.
+
+`mrs-developer` is included and installed by default when you generate a project with Cookieplone.
+Use the following command to install the configuration of `mrs.developer.json` in your project.
+
+```shell
+make install
+```
+
+
+## Configure `mrs-developer`
+
+{file}`mrs.developer.json` is the configuration file that instructs `mrs-developer` from where it should pull the packages.
+Cookieplone includes an empty one for you.
+Edit {file}`mrs.developer.json` and add the following code.
+
+```json
+{
+  "acme-volto-foo-addon": {
+    "output": "packages",
+    "package": "@acme/volto-foo-addon",
+    "url": "git@github.com:acme/my-volto-addon.git",
+    "path": "src"
+  }
+}
+```
+
+Then run:
+
+```bash
+make install
+```
+
+Now the add-on appears in `packages/acme-volto-foo-addon/`.
+
+```{note}
+The `package` property is optional.
+Use it only if your package has a namespace.
+
+`src` is required if the content of your add-on is located in the `src` directory.
+Since that is the convention for all Volto add-on packages, you must always include it.
```

Comment:
The above two lines seem odd to me because the add-on ends up in `packages/acme-volto-foo-addon/` instead of `src/addons/` as in Volto 17. Are these two lines needed or correct?

Reply:
Yes, in the past, this was needed but we adopted the standard in the monorepos (`packages`) so it's fine. It's referring to where the code is inside the add-on, not in the project.